testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/saml/IdpInitiatedLoginTest.java

```diff
@@ -184,4 +185,19 @@ public void testIdpInitiatedLoginWithOIDCClient() {
         adminClient.realm(REALM_NAME).clients().get(clientRep.getId()).update(ClientBuilder.edit(clientRep)
                 .protocol(SamlProtocol.LOGIN_PROTOCOL).build());
     }
+
+    @Test
+    public void testSamlIdPPostBindingPage() throws Exception {
+        new SamlClientBuilder()
+                .idpInitiatedLogin(getAuthServerSamlEndpoint(REALM_NAME), "sales-post").build()
+                .login().user(bburkeUser).build()
+                .execute(r -> {
+                    Assert.assertThat(r, statusCodeIsHC(Response.Status.OK));
+                    Assert.assertThat(r, bodyHC(allOf(
+                            containsString("Redirecting to identity provider, please wait."),
```

Comment:
```suggestion
                            containsString("Redirecting back to application, please wait."),
```

Reply:
The text is obtained from https://github.com/keycloak/keycloak/pull/6045/files#diff-b5d1ca446863b2174e6d69b58e2b3b2fR324 

Do we want to introduce two strings? One for SAMLResponse and the other for SAMLRequest?